### PR TITLE
replace /api/entities?action=changes by a generalized version of /api/entities?action=contributions

### PR DESCRIPTION
**Pull request by maxlath**
needed by the client for https://github.com/inventaire/inventaire-client/pull/239

### CODE REVIEW

File: server/controllers/entities/contributions.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const responses_ = __.require('lib', 'responses')
 const patches_ = require('./lib/patches')
 
 const sanitization = {
-  user: {},
+  user: { optional: true },
   limit: { default: 100, max: 10000 },
```
Comment (jum-s) [inline server/controllers/entities/contributions.js:10]: maybe 10 000 is too much. It made sense for a single entity, but for all of them, that's only a fraction of the exiting changes and, at the same time, this page could quickly become quite intense for the server to load all those changes.
bonus point : define recent in the branch name `recent-changes` :)

Comment (maxlath) [reply server/controllers/entities/contributions.js:10]: @jum-s how much then? 1000?

Comment (jum-s) [reply server/controllers/entities/contributions.js:10]: there is a reason why i did not suggest any figure. I guess 1000 is fine

Comment (maxlath) [reply server/controllers/entities/contributions.js]: done in 52b7368e

**Pull request merged**